script/Arcueid/Day 5/05_00_ARC05_4.txt

```diff
@@ -307,7 +307,8 @@ It's just like she says... but...
 [sha:46799c69025e33f7954627edc3f217785a6e6548]{
 -- Page 338, Offset 9457.
 -- 納得がいったのか、ふーん、とアルクェイドは他人事のように流していた。
-Whether she understood or not, Arcueid makes a noncommittal sound and drops the topic.
+//idk if this is good enough, but i definitely dont love the original
+I don't know if she's satisfied or not, but she gives a vague reply and drops the topic.
```

Comment:
I actually _like_ the original and would suggest a revert. It's quite natural imo

Reply:
I'd prefer them to be merged, something like:

> I don't know if she understood or not, but she makes a noncommittal sound and drops the topic.